### DISCUSSION TITLE
Tabs don't print properly

### DISCUSSION THREAD

**Discussion by 8008man** [thonny/thonny#3223] [General]
I started to use Thonny for code development on a Raspberry Pi-PICO board via MicroPython.  When I use tabs to align comments in a vertical line, the printed version has the comments spread hither and yon.  I've never seen an IDE do that.  Is there a setting to control how the "tabbed" comments appear?  I save listings in a lab notebook and now I must paste code in an agnostic editor and reposition comments so they fit on a printed page.  Am I missing something obvious?  --Jon

**Comment by aivarannamaa** [comment]
Thonny 5.0 will come with some changes in how tabs are displayed. Maybe this will fix your issue as well.
